modules/unsupported/geojson-core/src/main/java/org/geotools/data/geojson/ObjectMapperFactory.java

```diff
@@ -43,4 +43,11 @@ class ObjectMapperFactory {
     public static ObjectMapper getDefaultMapper() {
         return DEFAULT_MAPPER;
     }
+
+    public static ObjectMapper getDefaultMapper(int maxDecimals) {
+        // I can't work out how to deregister a module from the mapper so make a new one.
```

Comment:
It's not possible to un-/ deregister modules completely, creating a new mapper is the only way